lib/preview.js

```diff
@@ -210,7 +213,13 @@ const preview = async (
     PORT,
     protocol,
   };
-
+  const ungated = await isUngatedForPreview(sessionInfo);
+  if (!ungated) {
+    console.log(
+      `Portal ${accountId} is missing a required gate for this feature.`
```

Comment:
Since this will be public facing -- I wonder if we should omit the term "gate" since that is something internal and instead go with phrasing like:

`Portal ${accountId} doesn't have the correct permissions to use this feature.`

Is there a way log this in the console and log something regarding the gate to the debug log instead? `Portal ${accountId} is missing a required gate for this feature.` or even inserting the name of the missing gate here in order to identify easier. `Portal ${accountId} is missing a required gate - ${gateName} - which is required for this feature.`

Thoughts?

Reply:
Hiding the term gate seems fine to me!

Logging the gate name I'm meh about - there's only one for this feature so if the command crashes with this it's very obvious which one is missing. And we should eventually be ungated to all so the impact of the difference is low.

Reply:
Ah, yes, I was initially thinking about extensibility of the "gating" feature similar to the above -- so that in future implementations of gate checks in various parts of the CLI there would be a DRY error logger that could be used to expose the gate name causing the issue. BUT! This is something that can live outside of the scope of this particular project/PR -- so I can make an issue to follow up on gating logic in the CLI in order to revisit later.